### ISSUE TITLE
Generate yaml for postcoordination properties

### ISSUE THREAD

**Issue by tudorache** [who-icatx/whofic-models#11]
The postcoordination properties are needed to create the logical definitions. They are already extracted in the OWL file referenced below. We can include this OWL file directly in the content model, or generate the yaml, and then regenerate the OWL file..

https://github.com/who-icatx/content-migration/blob/main/script/template/icd-base.owl

**Comment by samsontu** [comment]
Yes, this past week I was trying to figure out how to add these properties. Initially I was using the ICD Schema's list of post-coordination properties. Thanks for the suggestion to use the declarations in the icd-base.owl. However, I found that the icd:Schema has additional post-coordination properties like jointInvolvementInFracture, extentOfBurnByBodySurface. I added them at the end. I used LinkML subproperty_of to indicate sub property relationship. However, the generated OWL ontology flatten everything. I still need to follow up on this.

**Comment by samsontu** [comment]
The following post-coordination axes are specified in the ICD Schema but not in the list object properties in the whofic-2024-01-21.owl file: 

- extentOfBurnByBodySurface    
- extentOfFullThicknessBurnByBodySurface
- outcomeOfFullThicknessBurn
- occupationalDescriptor      
- sportsActivityDescriptor
- transportEventDescriptor
- violenceDescriptor

**Comment by tudorache** [comment]
The postcoordination axes listed in the schema should be part of the OWL file. Not sure why they were not part of the export, maybe they were not used.

I've attached the hierarchy of all postcoordination axes in a tab-indented file:

[postcoordination_axes_hierarchy.txt](https://github.com/user-attachments/files/15616287/postcoordination_axes_hierarchy.txt)
